### ISSUE TITLE
ESLINT

### ISSUE THREAD

**Issue by adammunoz** [rakuten-frontend/rakuten-react-kit#24]
@rogeriotaques has reported:
"I've noticed React Kit has aslant set in the package.json, but I couldn't find it been used anywhere! Like, there's no settings to run it neither at package.json or webpack.config"

also:

"when lint is set, lots of errors and warnings are coming from source-code"

**Comment by rogeriotaques** [comment]
Hey, @adammunoz !

Thanks for create an issue for that.

Just in case, here's the `.eslintrc` I'm using here.

```
{
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "env": {
    "browser": true,
    "node": true
  },
  "extends": ["airbnb-base"],
  "parser": "babel-eslint",
  "plugins": ["react"],
  "settings": {
    "react": {
      "createClass": "createReactClass", // Regex for Component Factory to use, default to "createReactClass"
      "pragma": "React",  // Pragma to use, default to "React"
      "version": "15.0" // React version, default to the latest React stable release
    }
  },
  "rules": {
    "react/jsx-uses-react": "error",
    "react/jsx-uses-vars": "error",
    "space-before-function-paren": ["error", "always"],
    "comma-dangle": ["error", "only-multiline"],
    "import/no-unresolved": "off",
    "import/no-extraneous-dependencies": "off",
    "import/extensions": ["error", "never", {"svg": "always", "scss": "always", "css": "always"}],
    "padded-blocks": "off",
    "space-in-parens": "off",
    "max-len": ["error", 120]
  }
}
```

Note that it's using ESlint 3.x and Webpack 2.

**Comment by linmic** [comment]
I'd recommend extend from [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb). To do this we can learn from the best and of course we can tweak it according to our own opinions.

**Comment by adammunoz** [comment]
@linmic please go ahead :-)

**Comment by adammunoz** [comment]
Solved by #36
